src/thimble_server/data_access_layer/sql/select/select_post.sql

```diff
@@ -0,0 +1,2 @@
+-- selects post information for a given post ID
+SELECT file, poster FROM posts WHERE id = :postid;
```

Comment:
in the future can we adjust the query to select based on different types of parameters? So the same function could select by postid or audio hash?

Maybe these should just be separate